client/my-sites/domains/domain-management/edit/wpcom-domain.jsx

```diff
@@ -55,7 +55,43 @@ class WpcomDomain extends React.Component {
 		} );
 	};
 
-	getEditSiteAddressBlock() {
+	handlePickCustomDomainClick = () => {
+		const { domain } = this.props;
+		const domainType = getDomainTypeText( domain );
+
+		this.props.recordGoogleEvent(
+			'Domain Management',
+			`Clicked "Pick a Custom Domain" navigation link on a ${ domainType } in Edit`,
```

Comment:
> Pick a Custom Domain

The upper case Custom Domain isn't necessary. Can we make it all lower case "Pick a custom domain?"